Formula/groonga.rb

```diff
@@ -41,6 +43,11 @@ class Groonga < Formula
     sha256 "bc83d1e5e0f32d4b95e219cb940a7e3f61f0f743abd3bd47c2d436a34e503870"
   end
 
+  resource "stemmer" do
+    url "https://github.com/snowballstem/snowball.git",
+      :revision => "71936098048f915a797de990776564c924d40b6a"
```

Comment:
Ubuntu, CentOS, etc. all have libstemmer packages which I believe are based on the 2014 source tar, which is certainly better than nothing, so I could alter the PR to use that if necessary.  groonga/groonga#430 suggests that the upstream project will eventually integrate libstemmer similarly to my implementation here.